### ISSUE TITLE
Clippings Manager: Error when attempting to assign or change shortcut key that is already being used

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#39]
In Clippings Manager, if there is a clipping defined with a placeholder with a bad default value (e.g. the default value definition is missing a closing curly brace), then when I try to assign the clipping a shortcut key, nothing happens.

Example placeholder content:
> The quick brown fox $[verb{jumps}] over the $[adjective{lazy] dog.

And the error from the browser console:
```
NS_ERROR_FAILURE: Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) 
[nsIStringBundle.GetStringFromName] . . . . XStringBundle:21:0
```

**Comment by aecreations** [comment]
The placeholder syntax error has NOTHING to do with this bug.  This bug is caused by a reference to a non-existent string key in the clippings.properties file.  Bug title has been updated.

**Comment by aecreations** [comment]
Fixed in Clippings 5.0 beta 2
